x-pack/plugins/cross_cluster_replication/public/app/constants/form_schemas.js

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
I deleted this file in the advanced settings PR, so it shouldn't be needed any more.